nuxeo-admin-console-web/angular-app/src/app/features/bulk-action-monitoring/components/bulk-action-monitoring-form/bulk-action-monitoring-form.component.html

```diff
@@ -14,12 +14,12 @@ <h1 *ngIf="pageTitle" id="page-title" tabindex="0" aria-live="polite">
         <button mat-flat-button color="primary" type="submit" class="bulk-action-form__button"
             [disabled]="isBulkActionBtnDisabled">
             {{ BULK_ACTION_LABELS.BULK_ACTION_BUTTON_LABEL }}
-        </button> <br><br>
-        <ng-container *ngIf="bulkActionResponse && bulkActionResponse?.commandId">
-            <div *ngFor="let item of bulkActionResponse | keyvalue">
-                <div>{{item.key}} ------- {{item.value}}</div>
-            </div>
+        </button>
+        <div class="bulk-action-container__result" *ngIf="bulkActionResponse && bulkActionResponse?.commandId">
+            <hr>
+            <bulk-action-monitoring-result
```

Comment:
As discussed, please create a bulk action monitoring container which would have both Form and Result components. Form component will be shown everytime, and result component will be visible conditionally.